frontend/packages/kubevirt-plugin/src/components/vm-status/vm-status.tsx

```diff
@@ -102,6 +115,11 @@ export const VMStatus: React.FC<VMStatusProps> = ({
   )}`; // to default tab
   const additionalText = verbose ? getAdditionalImportText(statusDetail.pod) : null;
 
+  const unpauseVMI = async (event) => {
```

Comment:
missing type for event
```suggestion
  const unpauseVMI = async (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```

Reply:
Done!